documentation/milestones/individual_milestones/communication_platforms.md

```diff
@@ -60,31 +114,63 @@ Any code used to deploy or integrate these services should be made open source.
+- [ ] Chat is live with the following conditions satisfied:
+  - [ ] Users can post messages to public channels (potentially with some restrictions applied).
+  - [ ] Users can send private messages to each other as well as view private and public channels.
+  - [ ] Users can reply to messages in a thread that other users can see.
+  - [ ] Users can use emojis to react to messages.
+- [ ] Wiki is live with the following conditions satisfied:
+  - [ ] At least 1 document has been added to the wiki.
+  - [ ] Without logging in, members of the public can see wiki documents.
+  - [ ] Users can see history of changes to all documents.
+  - [ ] Web analytics have been integrated into the wiki and administrators can report on unique users, page views, etc.
+- [ ] Ticketing system is live with the following conditions satisfied:
+  - [ ] Without logging in, members of the public can see tickets that are being worked on.
+  - [ ] Default 'templates' have been created for tickets, epics, etc, that have the fields we've decided are necessary for the project.
+  - [ ] Users can create epics (or a similar concept) and track burndown.
+  - [ ] Members of the public can submit feature requests or report bugs
+- [ ] Calendar is live with the following conditions satisfied:
+  - [ ] At least 1 event has been added
+  - [ ] Without logging in, members of the public can see calendar events that are open to the public
+  - [ ] Members of the public can copy events to their own calendar and/or RSVP to attend the event
+- [ ] Email listserv is live with the following conditions satisfied
+  - [ ] At least one email has been sent out to the listserv
+  - [ ] Any member of the listserv can send emails to the entire listserv
+  - [ ] Any member of the listserv can reply all to an email
+  - [ ] Any member of the listserv can reply directly to an email sender
+- [ ] Shared email inbox is live with the following conditions satisfied
+  - [ ] At least 1 email has been received
 
 ### Proposed metrics for measuring goals/value/definition of done
 
-1. Number of monthly active users in Chat, total
-2. Number of monthly active users in wiki, total
-3. Number of monthly active users in Chat, external
-4. Number of monthly active users in wiki, external
-5. Weekly volume of Chat messages
-6. Weekly volume of wiki edits/contributions
+TODO: See if we can reduce the list of metrics to track
+
+1. Chat Metrics
+   1. Number of monthly active users in chat, total
+   2. Number of monthly active users in chat, external
+   3. Weekly volume of Chat messages
+2. Wiki Metrics
+   1. Number of monthly active users in wiki, total
+   2. Number of monthly active users in wiki, visitors
+   3. Weekly volume of wiki edits/contributions
+3. Calendar Metric: Number of external users attending public meetings
+4. Listserv Metrics
+   1. Number of subscribed users in listserv, total
+   2. Number of subscribed users in listserv, external
+   3. Average number of responses in listserv thread
```

Comment:
I'm thinking a bit on what this metric implies. Is it good to have many responses, or few? 

Do we want to measure instead total volumes of listserve traffic, with high being seen as a good thing? On the other hand, it's not good if everyone's emailing us "hey the API is broken." Hmm.

Reply:
@lucasmbrown-usds This is a good question and one that I was debating when I added this metric. I opted for average number of responses to a thread instead of total traffic for a few reasons:

- Threads with a lot of activity in the CEJST google group seemed to be better indications of user engagement, in contrast to say several separate message threads with no responses.
- Total traffic could still be high if lots of individuals were messaging with questions or issues but never get a response to their initial thread.

Perhaps the best strategy here is to measure both total traffic and responses per message thread with the goal of having a high number of active threads with meaningful engagement per thread?